build-system/tasks/dist.js

```diff
@@ -129,6 +130,9 @@ async function dist() {
       compileAllJs(options),
     ]);
   }
+  if (argv.fortesting) {
+    await buildBentoE2E(options);
+  }
```

Comment:
Just checking, is this skipped if e2e tests are run with the `--nobuild` flag?

Reply:
Nope, I currently don't understand how the ci caching works so it failed when when `--nobuild` did not build it. I might need your help if we want that functionality.

Reply:
AFAIK we usually don't gate building anything on `fortesting`. That flag is exclusively used as a runtime option for bundled code (i.e. to strip devAsserts)